src/main/java/com/github/manolo8/darkbot/core/manager/HeroManager.java

```diff
@@ -196,13 +196,19 @@ public boolean attackMode() {
 
     public boolean attackMode(Npc target) {
         if (target == null) return attackMode();
-        Config.ShipConfig config = this.main.config.GENERAL.OFFENSIVE;
+        Config.ConfigFormation config = this.main.config.GENERAL.OFFENSIVE;
```

Comment:
This datatype change being needed points to a problem; ideally we do not want plugins to need to change how they do any of their work for this, the Config.ShipConfig type ideally should work (ie: just don't make it a new datatype, update the existing one)